pyscal/pyscallist.py

```diff
@@ -68,6 +69,104 @@ def append(self, pyscal_obj):
             raise ValueError
         self.pyscal_list.append(pyscal_obj)
 
+    def df(self):
```

Comment:
should we have optional input for SLGOF (SL instead of SG) , and family 2? ok to expand later of course

Reply:
Good point. SLGOF is quite trivial to add, family 2 should be possible, but should include some refactoring of the code making the sof3 table in WaterOilGas, for it to be reused here.

Reply:
could add the SLGOF flag for now which is pretty straight forward, and delay family 2?